knowledge/laws/traffic_laws_US/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: Chris Denton
```

Comment:
Please put your GitHub Username here.

Reply:
Changed to GitHub username and saved! :)